src/Common/DatabaseAccountUtility.ts

```diff
@@ -15,3 +17,9 @@ function isPrivateEndpointConnectionsEnabled() {
 export function isPublicInternetAccessAllowed(): boolean {
   return !isVirtualNetworkFilterEnabled() && !isIpRulesEnabled() && !isPrivateEndpointConnectionsEnabled();
 }
+
+export function getWorkloadType(): WorkloadType {
```

Comment:
What does this return when no workload tag is present ? Can you please add unit tests for this ? Maybe we need a 'None' workload type enum ?